### ISSUE TITLE
Resizing event is not fired for config nodes

### ISSUE THREAD

**Issue by hassony2** [node-red/node-red#894]
When creating a custom config node and adding a oneditresize function, the function is not triggered when the node editor is resized.

For instance, when adding the following code to the html file of the custom config node

```javascript
    oneditresize: function(size) {
        console.log('resizing detected');
    }
```
resizing the dialog editor  doesn't trigger the browser's console logs. The same code works just fine with non-config custom nodes.

Running in google chrome
Node-red version : 0.13.4
Node version : 4.3.2

**Comment by knolleary** [comment]
Hi, thanks for raising.

This is already fixed in 0.14 which is due out next week.

**Comment by knolleary** [comment]
Closing this issue as we finalise 0.14.0... any day now.
